src/services/remote-config/RemoteConfig.ts

```diff
@@ -27,7 +27,13 @@ export enum IntKeys {
   NOTIFICATION_POLLING_FREQ_MS = 'NOTIFICATION_POLLING_FREQ_MS'
 }
 
-export enum BooleanKeys {}
+export enum BooleanKeys {
+  /**
+   * If the optimized trending track ids endpoint should be used for badges
+   * else the trending tracks endpoint will be used
```

Comment:
nit, maybe link the protocol PR or release SHA or something to know when it's safe to deprecate